2024-07-30 11:16:16,575 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-07-30 11:16:16,579 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-30 11:40:31,161 ERROR libqtile manager.py:spawn():L1297 couldn't find `pcmafm`
2024-07-30 11:40:38,574 ERROR libqtile manager.py:spawn():L1297 couldn't find `pcmanfm`
2024-07-30 11:44:02,154 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-30 11:44:33,588 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-30 11:47:58,364 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-30 11:51:03,247 ERROR libqtile base.py:on_done():L858 Failed to reschedule timer for net.
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 852, in on_done
    self.update(result)
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 756, in update
    old_width = self.layout.width
                ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/base/drawer.py", line 429, in width
    return self.layout.get_pixel_size()[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/pangocffi.py", line 134, in get_pixel_size
    pango.pango_layout_get_pixel_size(self._pointer, width, height)
TypeError: initializer for ctype 'PangoLayout *' must be a cdata pointer, not NoneType
2024-07-30 11:51:03,249 ERROR libqtile base.py:on_done():L858 Failed to reschedule timer for batterywidget.
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 852, in on_done
    self.update(result)
  File "/home/abhi/.config/qtile/battery.py", line 41, in update
    self.text = text
    ^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 523, in text
    self.layout.text = self.formatted_text
    ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/base/drawer.py", line 421, in text
    self.layout.set_attributes(attrlist)
  File "/usr/lib/python3.12/site-packages/libqtile/pangocffi.py", line 114, in set_attributes
    pango.pango_layout_set_attributes(self._pointer, attrs)
TypeError: initializer for ctype 'PangoLayout *' must be a cdata pointer, not NoneType
2024-07-30 11:51:03,249 ERROR libqtile base.py:on_done():L858 Failed to reschedule timer for memory.
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 852, in on_done
    self.update(result)
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 756, in update
    old_width = self.layout.width
                ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/base/drawer.py", line 429, in width
    return self.layout.get_pixel_size()[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/pangocffi.py", line 134, in get_pixel_size
    pango.pango_layout_get_pixel_size(self._pointer, width, height)
TypeError: initializer for ctype 'PangoLayout *' must be a cdata pointer, not NoneType
2024-07-30 11:51:03,250 WARNING libqtile lifecycle.py:_atexit():L33 Restarting Qtile with os.execv(...)
2024-07-30 11:51:19,388 ERROR libqtile manager.py:spawn():L1297 couldn't find `code`
2024-07-30 12:01:38,388 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-30 14:21:17,740 ERROR libqtile hook.py:fire():L193 Error in hook client_killed
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/hook.py", line 191, in fire
    i(*args, **kwargs)
  File "/usr/lib/python3.12/site-packages/libqtile/dgroups.py", line 256, in _del
    if group.persist:
       ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'persist'
2024-07-30 14:48:41,518 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-30 20:28:02,965 ERROR libqtile manager.py:spawn():L1297 couldn't find `pcmanfm`
2024-07-30 20:28:06,773 ERROR libqtile manager.py:spawn():L1297 couldn't find `pcmanfm`
2024-07-30 20:30:53,486 ERROR libqtile base.py:on_done():L858 Failed to reschedule timer for net.
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 852, in on_done
    self.update(result)
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 756, in update
    old_width = self.layout.width
                ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/base/drawer.py", line 429, in width
    return self.layout.get_pixel_size()[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/pangocffi.py", line 134, in get_pixel_size
    pango.pango_layout_get_pixel_size(self._pointer, width, height)
TypeError: initializer for ctype 'PangoLayout *' must be a cdata pointer, not NoneType
2024-07-30 20:30:53,487 ERROR libqtile base.py:on_done():L858 Failed to reschedule timer for memory.
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 852, in on_done
    self.update(result)
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 756, in update
    old_width = self.layout.width
                ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/base/drawer.py", line 429, in width
    return self.layout.get_pixel_size()[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/pangocffi.py", line 134, in get_pixel_size
    pango.pango_layout_get_pixel_size(self._pointer, width, height)
TypeError: initializer for ctype 'PangoLayout *' must be a cdata pointer, not NoneType
2024-07-30 20:30:53,489 WARNING libqtile lifecycle.py:_atexit():L33 Restarting Qtile with os.execv(...)
2024-07-30 20:31:02,511 WARNING libqtile lifecycle.py:_atexit():L33 Restarting Qtile with os.execv(...)
2024-07-30 22:09:03,563 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-31 09:41:02,079 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-31 10:14:55,779 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-31 10:18:15,460 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-31 10:29:04,971 ERROR libqtile base.py:on_done():L858 Failed to reschedule timer for net.
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 852, in on_done
    self.update(result)
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 756, in update
    old_width = self.layout.width
                ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/base/drawer.py", line 429, in width
    return self.layout.get_pixel_size()[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/pangocffi.py", line 134, in get_pixel_size
    pango.pango_layout_get_pixel_size(self._pointer, width, height)
TypeError: initializer for ctype 'PangoLayout *' must be a cdata pointer, not NoneType
2024-07-31 10:29:04,972 ERROR libqtile base.py:on_done():L858 Failed to reschedule timer for memory.
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 852, in on_done
    self.update(result)
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 756, in update
    old_width = self.layout.width
                ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/base/drawer.py", line 429, in width
    return self.layout.get_pixel_size()[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/pangocffi.py", line 134, in get_pixel_size
    pango.pango_layout_get_pixel_size(self._pointer, width, height)
TypeError: initializer for ctype 'PangoLayout *' must be a cdata pointer, not NoneType
2024-07-31 10:29:04,972 ERROR libqtile base.py:on_done():L858 Failed to reschedule timer for batterywidget.
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 852, in on_done
    self.update(result)
  File "/home/abhi/.config/qtile/battery.py", line 41, in update
    self.text = text
    ^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/widget/base.py", line 523, in text
    self.layout.text = self.formatted_text
    ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/base/drawer.py", line 421, in text
    self.layout.set_attributes(attrlist)
  File "/usr/lib/python3.12/site-packages/libqtile/pangocffi.py", line 114, in set_attributes
    pango.pango_layout_set_attributes(self._pointer, attrs)
TypeError: initializer for ctype 'PangoLayout *' must be a cdata pointer, not NoneType
2024-07-31 10:29:04,973 WARNING libqtile lifecycle.py:_atexit():L33 Restarting Qtile with os.execv(...)
2024-07-31 10:29:10,626 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-31 10:31:12,488 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-31 10:35:16,669 ERROR libqtile manager.py:spawn():L1297 couldn't find `betterlockscreen`
2024-07-31 10:35:19,117 ERROR libqtile manager.py:spawn():L1297 couldn't find `betterlockscreen`
2024-07-31 10:35:19,384 ERROR libqtile manager.py:spawn():L1297 couldn't find `betterlockscreen`
2024-07-31 10:35:19,576 ERROR libqtile manager.py:spawn():L1297 couldn't find `betterlockscreen`
2024-07-31 10:35:19,708 ERROR libqtile manager.py:spawn():L1297 couldn't find `betterlockscreen`
2024-07-31 10:35:19,855 ERROR libqtile manager.py:spawn():L1297 couldn't find `betterlockscreen`
2024-07-31 10:35:20,033 ERROR libqtile manager.py:spawn():L1297 couldn't find `betterlockscreen`
2024-07-31 12:44:51,378 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-31 17:57:47,991 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-31 21:49:39,141 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-07-31 21:49:39,160 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-07-31 22:24:08,719 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-07-31 22:24:08,737 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 10:07:53,612 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 10:43:41,632 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 10:45:39,595 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 10:45:39,595 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 10:45:39,596 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 10:45:39,620 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 10:48:21,366 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 10:48:21,368 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 10:48:21,378 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 10:56:39,206 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 10:56:39,217 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:02:17,963 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 11:02:17,967 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:02:17,986 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:23:28,085 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 11:23:28,086 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:23:28,095 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:23:44,744 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:23:44,749 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:26:48,976 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:26:48,981 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 11:26:48,982 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:26:48,987 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:27:09,217 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:27:09,223 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:30:24,942 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:30:24,950 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:30:45,213 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:30:45,213 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 11:30:45,214 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:30:45,223 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:37:05,734 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:37:05,741 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:49:01,592 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:49:01,598 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:52:51,044 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:52:51,049 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:53:00,396 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 11:53:00,397 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:53:00,397 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:53:00,402 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 11:55:18,931 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 11:55:18,936 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 12:01:57,739 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 12:03:19,157 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 12:03:19,166 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 12:03:30,710 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 12:03:30,715 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 12:03:30,715 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 12:03:30,719 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 12:09:58,952 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 12:09:58,956 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 12:09:58,957 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 12:09:58,964 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 18:15:46,863 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 18:15:46,870 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 18:15:46,871 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 18:15:46,951 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 21:44:47,553 ERROR libqtile hook.py:fire():L193 Error in hook client_killed
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/hook.py", line 191, in fire
    i(*args, **kwargs)
  File "/usr/lib/python3.12/site-packages/libqtile/dgroups.py", line 256, in _del
    if group.persist:
       ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'persist'
2024-08-01 22:39:52,257 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 22:39:52,257 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 22:39:52,258 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 22:39:52,283 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-01 23:15:18,779 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 23:15:18,784 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-01 23:15:18,784 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-01 23:15:18,792 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-02 16:27:04,559 WARNING libqtile lifecycle.py:_atexit():L33 Restarting Qtile with os.execv(...)
2024-08-02 18:00:19,425 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-02 18:00:19,429 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-02 18:00:19,430 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-02 18:00:19,454 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-02 18:53:06,592 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-02 18:53:06,596 WARNING libqtile core.py:_xpoll():L353 Shutting down due to disconnection from X server
2024-08-02 18:53:06,596 WARNING libqtile core.py:graceful_shutdown():L900 Server disconnected, couldn't close windows gracefully.
2024-08-02 18:53:06,610 WARNING libqtile lifecycle.py:_atexit():L37 Qtile will now terminate
2024-08-03 16:28:25,921 ERROR libqtile hook.py:fire():L193 Error in hook client_killed
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/hook.py", line 191, in fire
    i(*args, **kwargs)
  File "/usr/lib/python3.12/site-packages/libqtile/dgroups.py", line 256, in _del
    if group.persist:
       ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'persist'
2024-08-03 16:30:25,094 ERROR libqtile hook.py:fire():L193 Error in hook client_killed
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/hook.py", line 191, in fire
    i(*args, **kwargs)
  File "/usr/lib/python3.12/site-packages/libqtile/dgroups.py", line 256, in _del
    if group.persist:
       ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'persist'
2024-08-03 16:30:25,103 ERROR libqtile loop.py:_handle_exception():L62 Exception in event loop:
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/widget/bluetooth.py", line 389, in _config_async
    await self._connect()
  File "/usr/lib/python3.12/site-packages/libqtile/widget/bluetooth.py", line 393, in _connect
    self.bus = await MessageBus(bus_type=BusType.SYSTEM).connect()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/dbus_next/aio/message_bus.py", line 149, in connect
    await self._authenticate()
  File "/usr/lib/python3.12/site-packages/dbus_next/aio/message_bus.py", line 390, in _authenticate
    response = self._auth._receive_line(await self._auth_readline())
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/dbus_next/aio/message_bus.py", line 376, in _auth_readline
    buf += await self._loop.sock_recv(self._sock, 2)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/asyncio/selector_events.py", line 391, in sock_recv
    handle = self._add_reader(fd, self._sock_recv, fut, sock, n)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/asyncio/selector_events.py", line 288, in _add_reader
    self._selector.modify(fd, mask | selectors.EVENT_READ,
  File "/usr/lib/python3.12/selectors.py", line 389, in modify
    self._selector.modify(key.fd, selector_events)
FileNotFoundError: [Errno 2] No such file or directory
2024-08-03 16:30:26,160 ERROR libqtile hook.py:fire():L193 Error in hook client_killed
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/hook.py", line 191, in fire
    i(*args, **kwargs)
  File "/usr/lib/python3.12/site-packages/libqtile/dgroups.py", line 256, in _del
    if group.persist:
       ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'persist'
2024-08-03 16:33:28,957 ERROR libqtile hook.py:fire():L193 Error in hook client_killed
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/hook.py", line 191, in fire
    i(*args, **kwargs)
  File "/usr/lib/python3.12/site-packages/libqtile/dgroups.py", line 256, in _del
    if group.persist:
       ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'persist'
2024-08-03 16:35:18,197 ERROR libqtile hook.py:fire():L193 Error in hook client_killed
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/hook.py", line 191, in fire
    i(*args, **kwargs)
  File "/usr/lib/python3.12/site-packages/libqtile/dgroups.py", line 256, in _del
    if group.persist:
       ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'persist'
2024-08-03 16:35:19,869 ERROR libqtile base.py:function():L342 Exception calling "<function restore_minimized_window at 0x7b1f8ff2c2c0>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: restore_minimized_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:20,383 ERROR libqtile base.py:function():L342 Exception calling "<function restore_minimized_window at 0x7b1f8ff2c2c0>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: restore_minimized_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:20,579 ERROR libqtile base.py:function():L342 Exception calling "<function restore_minimized_window at 0x7b1f8ff2c2c0>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: restore_minimized_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:20,751 ERROR libqtile base.py:function():L342 Exception calling "<function restore_minimized_window at 0x7b1f8ff2c2c0>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: restore_minimized_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:21,034 ERROR libqtile base.py:function():L342 Exception calling "<function minimize_focused_window at 0x7b1f8ff2cd60>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: minimize_focused_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:21,211 ERROR libqtile base.py:function():L342 Exception calling "<function minimize_focused_window at 0x7b1f8ff2cd60>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: minimize_focused_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:21,378 ERROR libqtile base.py:function():L342 Exception calling "<function minimize_focused_window at 0x7b1f8ff2cd60>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: minimize_focused_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:21,549 ERROR libqtile base.py:function():L342 Exception calling "<function minimize_focused_window at 0x7b1f8ff2cd60>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: minimize_focused_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:21,720 ERROR libqtile base.py:function():L342 Exception calling "<function minimize_focused_window at 0x7b1f8ff2cd60>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: minimize_focused_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:22,200 ERROR libqtile base.py:function():L342 Exception calling "<function restore_minimized_window at 0x7b1f8ff2c2c0>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: restore_minimized_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:22,595 ERROR libqtile base.py:function():L342 Exception calling "<function minimize_focused_window at 0x7b1f8ff2cd60>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: minimize_focused_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:22,901 ERROR libqtile base.py:function():L342 Exception calling "<function minimize_focused_window at 0x7b1f8ff2cd60>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: minimize_focused_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:23,140 ERROR libqtile base.py:function():L342 Exception calling "<function restore_minimized_window at 0x7b1f8ff2c2c0>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: restore_minimized_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:23,374 ERROR libqtile base.py:function():L342 Exception calling "<function minimize_focused_window at 0x7b1f8ff2cd60>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: minimize_focused_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:23,589 ERROR libqtile base.py:function():L342 Exception calling "<function restore_minimized_window at 0x7b1f8ff2c2c0>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: restore_minimized_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:23,812 ERROR libqtile base.py:function():L342 Exception calling "<function minimize_focused_window at 0x7b1f8ff2cd60>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: minimize_focused_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:24,030 ERROR libqtile base.py:function():L342 Exception calling "<function restore_minimized_window at 0x7b1f8ff2c2c0>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: restore_minimized_window() takes 0 positional arguments but 1 was given

2024-08-03 16:35:24,242 ERROR libqtile base.py:function():L342 Exception calling "<function minimize_focused_window at 0x7b1f8ff2cd60>":
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 339, in function
    function(self, *args, **kwargs)
TypeError: minimize_focused_window() takes 0 positional arguments but 1 was given


TypeError: restore_minimized_window() takes 0 positional arguments but 1 was given

2024-08-03 16:36:56,036 ERROR libqtile hook.py:fire():L193 Error in hook client_killed
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/hook.py", line 191, in fire
    i(*args, **kwargs)
  File "/usr/lib/python3.12/site-packages/libqtile/dgroups.py", line 256, in _del
    if group.persist:
       ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'persist'
